releases.json

```diff
@@ -1 +1,8 @@
-{}
+{
+  "0.1.0": {
+    "hash": "/ipfs/QmYxMR2B7L6rca2FgrjfuMP6mysR3fE77PdPLze2Gt35Uc",
+    "uploadedTo": {
+      "dappnode": "Thu, 21 Dec 2023 12:17:43 GMT"
+    }
+  }
+}
```

Comment:
Was this already deployed? If so, how and by whom?

Reply:
Yes, its available on IPFS, I builded it for testing purposes with:
```shell
npx @dappnode/dappnodesdk build
```